metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -791,12 +925,20 @@ def _to_metadata_events(  # noqa: C901
             upstreams = []
             observed_lineage_ts = datetime.datetime.now(tz=datetime.timezone.utc)
             for view_ref in sorted(self.upstream_views):
+                # set file_path to ViewFieldType.UNKNOWN if file_path is not available to keep backward compatibility
+                # if we raise error on file_path equal to None then existing test-cases will fail as mock data doesn't have required attributes.
+                file_path: str = (
+                    cast(str, self.upstream_views_file_path[view_ref.include])
```

Comment:
why do we need this cast

Reply:
because self.upstream_views_file_path[view_ref.include] is optional, None check is there but still lint was raising error